network-area-diagram/src/main/java/com/powsybl/nad/model/TextPosition.java

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.nad.model;
+
+/**
+ * @author Massimo Ferraro {@literal <massimo.ferraro at soft.it>}
+ */
+public record TextPosition(Point position, Point connection) {
```

Comment:
Maybe a bit more explicit? and then it would be `getEdgeConnection` in the `TextNode`
```suggestion
public record TextPosition(Point topLeftPosition, Point edgeConnection) {
```

Reply:
parameters and methods renamed